operations/services.md

```diff
@@ -330,19 +313,24 @@ Delete availability blocks. Only block with no active [Availability update](#ava
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `AvailabilityBlocks` | array of [Availability block delete parameters](#availability-block-parameters) | required, max 1000 items | Availability blocks to be deleted. |
+| `ServiceId` | string | required | Unique identifier of the [Service](#service) to update. |
+| `AvailabilityUpdates` | array of [Availability update](#availability-update) | required, max 1000 items | Availability updates. |
 
-#### Availability block delete parameters
+#### Availability update
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
-| `Id` | string | required | Unique identifier of the availability block. |
-| `ServiceId` | string | required | Unique identifier of the [Service](#service). |
+| `StartUtc` | string | required | Start of the interval in UTC timezone in ISO 8601 format. |
+| `EndUtc` | string | required | End of the interval in UTC timezone in ISO 8601 format. |
+| `AvailabilityBlockId` | string | optional | Unique identifier of the [Availability block](#availability-block) whose availability to update. |
+| `ResourceCategoryId` | string | required | Unique identifier of the [Resource category](enterprises.md#resource-category) whose availability to update. |
+| `UnitCountAdjustment` | [Number update value](reservations.md#number-update-value) | required | Adjustment value to be applied on the interval, can be both positive and negative (relative adjustment, not an absolute number). If specified without `Value` parameter, removes all adjustments within the interval. |
```

Comment:
We should probably clarify the meaning in case of availability block id is specified. In order to keep it consistent, I guess you should still pass -5 if you want to reserve 5 rooms, correct? But then it probably shouldn't be possible to provide positive numbers which would correspond to availability blocks with negative size. Or should it be other way around and you should specify 5 if you want to reserve 5 and we should ban negative?